### ISSUE TITLE
Lists in Lists

### ISSUE THREAD

**Issue by jahelo** [cuba-platform/yarg#11]
Hi,

first i want to thank you for developing yarg it is a very nice idea and i like your approach of templating with office documents.

I have an Issue with Displaying Lists in Lists in an DOCX File. I'm trying to figure out what to put in the template to get it working. 

Let's take the invoice example. I want to create a report which contains multiple Invoices and puts the Invoicenumer over the corresponding list of items.

So basically i have the root band (level 1), a header band (level 2) , an Invoices band (level 2) and a positions band (level 3).

The documentation is missing this part and the sample "nestedbands" neither is very clear on it.

Could you provide some more information on how to achieve my goal with yarg? Is it even possible? 

Kind Regards

Jan

**Comment by faubertin-FA** [comment]
If I understand well your need, you want sub-bands ? If so, it seems not handled in DOCX template yet (from documentation : Horizontal bands in DOCX and DOC cannot contain sub-bands. If you need to use sub-bands, you would better use XLS(X) format.).

See https://doc.cuba-platform.com/reporting-latest/reporting.html#template_doc

**Comment by tinhol** [comment]
Hello.

@faubertin-FA is absolutely right,

YARG does not support nested bands in DOC/ODT/DOCX templates.

Please use XLS/XLSX/HTML for such cases.

Thank you for your feedback.

**Comment by kjordan2001** [comment]
Does YARG support nested bands for PDFs?  I have one working when exporting to XLSX but it doesn't seem to end up in the report if exporting to PDF.
